tests/models/whisper/test_modeling_whisper.py

```diff
@@ -351,6 +351,20 @@ def test_requires_grad_with_frozen_encoder(self):
             self.assertFalse(all(encoder_grads))
             self.assertTrue(all(decoder_grads))
 
+    def test_requires_grad_encoder_embed_positions(self):
+        config = self.model_tester.get_config()
+        for model_class in self.all_model_classes:
+            model = model_class(config)
+            encoder = model.get_encoder()
+            self.assertFalse(encoder.embed_positions.weight.requires_grad)
+
+    def test_encoder_sinusoidal_embed_positions(self):
```

Comment:
@sanchit-gandhi I have already added the test for sinusoidal embeddings here. Not done yet for Jax/TF though.

Reply:
Cool! Looks good!